lib/gooddata/models/automated_data_distribution.rb

```diff
@@ -8,6 +8,18 @@ module GoodData
   class AutomatedDataDistribution < Rest::Resource
     attr_writer :output_stage
 
+    def output_stage
```

Comment:
do not agree with this version. The old one in https://github.com/gooddata/gooddata-ruby/pull/791/files#diff-dad39358c99f5e64258fd991e1752949R33 is better.